draft-ietf-git-using-github.md

```diff
@@ -168,17 +171,16 @@ found.  For instance, the name could be ietf-wg-\<wgname>, as recommended in
 
 A single organization SHOULD NOT be used for all IETF activity, or all activity
 within an area. Large organizations create too much overhead for general
-management tasks, particularly when there is a need to maintain membership.
+management tasks.
 
-Each organization requires owners.  The owner team for a Working Group
-repository MUST include responsible Area Directors.  Area Directors MAY also
-designate a delegate that becomes an owner and Working Group chairs MAY also be
-owners.
+GitHub requires that each organization have at least one owner.  The owners for
+a Working Group repository MUST include responsible Area Directors.  Area
+Directors MAY also designate a delegate that becomes an owner and Working Group
+chairs MAY also be owners.
 
-A team with administrator access SHOULD be created for the Working Group Chairs
-and any Working Group Secretary.  Administrator access is preferable, since this
-does not also include the ability to push to all repositories and ownership does
-not grant any other significant privileges.
```

Comment:
I can understand that there's not a huge need to mention the details of Administrator privilege in these guidelines.

Reply:
Well, the point was that this was flat wrong, based on an outdated authorization model.

Reply:
Should we mention that the model (as is now documented) is also subject to change?